docs/src/main/sphinx/functions/setdigest.rst

```diff
@@ -0,0 +1,156 @@
+The similarity of any two sets is estimated by comparing their signatures.
+
+The Trino type for this data structure is called ``setdigest``.
+
+Trino offers the ability to merge multiple Set Digest data sketches.
+
+The following example showcases how the Set Digest functions are
+employed to estimate the similarity between texts::
+
+    WITH text_input(id, text) AS (VALUES
+                                 (1, 'The quick brown fox jumps over the lazy dog'),
+                                 (2, 'The quick red fox jumps over the lazy dog'),
+                                 (3, 'The quick and the lazy'),
+                                 (4, 'The quick brown fox jumps over the dog')),
+         text_ngrams(id, ngrams) AS (
+             SELECT id,
+                    transform(
+                      ngrams(
+                        split(text, ' '),
+                        4
+                      ),
+                      token -> array_join(token, ' ')
+                    )
+             FROM text_input
+         ),
+         minhash_digest(id, digest) AS (
+             SELECT id,
+                    (SELECT make_set_digest(v) FROM unnest(ngrams) u(v))
+             FROM text_ngrams
+         ),
+         setdigest_side_by_side(id1, digest1, id2, digest2) AS (
+             SELECT m1.id as id1,
+                    m1.digest as digest1,
+                    m2.id as id2,
+                    m2.digest as digest2
+             FROM (SELECT id, digest FROM minhash_digest) m1
+             JOIN (SELECT id, digest FROM minhash_digest) m2
+               ON m1.id != m2.id AND m1.id < m2.id
+        )
+    SELECT id1,
+           id2,
+           intersection_cardinality(digest1, digest2) AS intersection_cardinality,
+           jaccard_index(digest1, digest2)            AS jaccard_index
+    FROM setdigest_side_by_side
+    ORDER BY id1, id2;
+
+.. code-block:: text
+
+     id1 | id2 | intersection_cardinality |    jaccard_index
+    -----+-----+--------------------------+---------------------
+       1 |   2 |                        3 | 0.16666666666666666
+       1 |   3 |                        0 |                 0.0
+       1 |   4 |                        4 |                 0.6
+       2 |   3 |                        0 |                 0.0
+       2 |   4 |                        1 |                 0.0
+       3 |   4 |                        0 |                 0.0
+
+Serialization
+-------------
+
+Data sketches can be serialized to and deserialized from ``varbinary``. This
+allows them to be stored for later use.
+
+
+Functions
+---------
+
+.. function:: make_set_digest(x) -> setdigest
+
+    Composes all input values of ``x`` into a ``setdigest``.
+
+    ``x`` must be of ``bigint`` type.
+
+
+.. function:: merge_set_digest(setdigest) -> setdigest
+
+    Returns the ``setdigest`` of the aggregate union of the individual ``setdigest``
+    Set Digest structures.
+
+.. function:: cardinality(setdigest) -> long
+    :noindex:
```

Comment:
maybe add an anchor for this function so we can link to it anyway and then add it to the list page

Reply:
Can you please state the appropriate syntax for adding the anchor for the function definition?

Reply:
```
.. _setdigest-cardinality:
```

and then use 
```
:ref:`cardinality <setdigest-cardinality>`
```

Reply:
Thanks for the tip. I've added the anchor now before the `cardinality` function.